source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRImageRenderer.mm

```diff
@@ -48,22 +49,75 @@ - (CGSize) getImageSize:(std::shared_ptr<Image> const &) imgElem
+                                     attribute:NSLayoutAttributeCenterY
+                                    multiplier:1
+                                      constant:0]];
+
+    switch (alignment)
+    {
         case HorizontalAlignment::Center:
-            return UIViewContentModeCenter;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeCenterX
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeCenterX
+                                            multiplier:1
+                                              constant:0]];
+                return constraints;
+        }
         case HorizontalAlignment::Left:
-            return UIViewContentModeLeft;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeLeading
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeLeading
+                                            multiplier:1
+                                              constant:0]];
+            return constraints;
+        }
         case HorizontalAlignment::Right:
-            return UIViewContentModeRight;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeTrailing
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeTrailing
+                                            multiplier:1
+                                              constant:0]];
+            return constraints;
+        }
         default:
-            return UIViewContentModeLeft;
+        {
+            [constraints addObject: 
+                [NSLayoutConstraint constraintWithItem:superview
+                                             attribute:NSLayoutAttributeLeading
+                                             relatedBy:NSLayoutRelationEqual
+                                                toItem:view
+                                             attribute:NSLayoutAttributeLeading
+                                            multiplier:1
+                                              constant:0]];
+            return constraints;
+        }
     }
+    return constraints;
 }
-- (UIView* ) render: (UIStackView*) viewGroup
+-(UIView*)render: (UIView*) viewGroup
```

Comment:
i'm not aware of that, but i can do it with vim and sed if I can find the correct regex. 

---
In reply to: [128600681](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128600681) [](ancestors = 128600681)